core-web/libs/data-access/src/lib/dot-workflows-actions/dot-workflows-actions.service.ts

```diff
@@ -53,6 +58,36 @@ export class DotWorkflowsActionsService {
             .pipe(pluck('entity'));
     }
 
+    /**
+     * Returns the workflow actions of the passed url, hostId and language
+     *
+     * @param {string} host_id
+     * @param {string} language_id
+     * @param {string} url
+     * @param {DotRenderMode} [renderMode]
+     * @returns {Observable<DotCMSPageWorkflowState>}
+     * @memberof DotWorkflowsActionsService
+     */
+    getByUrl(
+        host_id: string,
+        language_id: string,
+        url: string,
+        renderMode?: DotRenderMode
+    ): Observable<DotCMSPageWorkflowState> {
```

Comment:
If we have more than two `params`, it's better to use an `object`.

```suggestion
    getByUrl({
        host_id,
        language_id,
        url,
        renderMode
    }: WorkflowRequestData): Observable<DotCMSPageWorkflowState> {
```